extension.json

```diff
@@ -1,59 +1,64 @@
 {
 	"name": "Page Approvals",
-
```

Comment:
We typically have this more verbose spacing in here to make the file a bit more readable.

Reply:
Oh. Okay. I was using professiona wiki xml for this and didn't see that inproadmin.